crates/wash/src/cli/cmd/up.rs

```diff
@@ -210,19 +210,19 @@ pub struct WasmcloudOpts {
     pub rpc_credsfile: Option<PathBuf>,
 
     /// An IP address or DNS name to use to connect to NATS for Control Interface (CTL) messages, defaults to the value supplied to --nats-host if not supplied
-    #[clap(long = "ctl-host", env = WASMCLOUD_CTL_HOST)]
+    #[clap(long = "nats-host",alias = "ctl-host", env = WASMCLOUD_NATS_CTL_HOST)]
     pub ctl_host: Option<String>,
 
     /// A port to use to connect to NATS for CTL messages, defaults to the value supplied to --nats-port if not supplied
-    #[clap(long = "ctl-port", env = WASMCLOUD_CTL_PORT)]
+    #[clap(long = "nats-port",alias = "ctl-port", env = WASMCLOUD_NATS_CTL_PORT)]
```

Comment:
```suggestion
    #[clap(long = "nats-ctl-port",alias = "ctl-port", env = WASMCLOUD_NATS_CTL_PORT)]
```
The long form arg should be modified here as well